server/src/routes/api/v1/destinationsReviewRouter.js

```diff
@@ -0,0 +1,19 @@
+import express from "express"
+import { Review, Destination } from "../../../models/index.js"
+
+const destinationReviewsRouter = new express.Router({ mergeParams: true})
+
+destinationReviewsRouter.get("/", async (req, res) => {
+    const { destinationId } = req.params
+    
+    try {
+        const destination = await Destination.query().findById(destinationId)
```

Comment:
we're doing the same logic here as in the destinations router when we get the one destination - add the $relatedQuery to that endpoint and delete this router entirely 🔥